index.html

```diff
@@ -16,7 +16,7 @@
 </script>
     <link rel="stylesheet" href="style1.css">
     <link rel="stylesheet" href="style2.css">
-    <title>portfolio</title>
+    <title>Mugisha's Portfolio</title>
```

Comment:
As suggested by the previous code reviewer:
* To make your site more accessible, your page title should adequately and briefly describe the content of the page (e.g. Mugisha's portfolio, Mugisha full-stack web developer).* 
Kindly fix this 🙏